more_itertools/more.py

```diff
@@ -1202,6 +1203,26 @@ def padded(iterable, fillvalue=None, n=None, next_multiple=False):
             yield fillvalue
 
 
+def repeat_last(iterable):
```

Comment:
What do you think of having a `default` value to emit endlessly, if *iterable* is empty?